commands/lxc2lxd_script.go

```diff
@@ -225,7 +225,7 @@ def convert_container(lxd_socket, container_name, args):
     # Load the container
     try:
         container = lxc.Container(container_name, args.lxcpath)
-    except:
+    except Exception:
```

Comment:
this was an upstream change in lxc-to-lxd, just shows up here because of the "go generate"